Doc/library/ctypes.rst

```diff
@@ -1178,9 +1178,9 @@ the root-object's underlying buffer.
 Another example that may behave different from what one would expect is this::
 
    >>> s = c_char_p()
-   >>> s.value = "abc def ghi"
+   >>> s.value = b"abc def ghi"
    >>> s.value
-   'abc def ghi'
+   b'abc def ghi'
    >>> s.value is s.value
    False
    >>>
```

Comment:
In order to help users not make the same mistake that the docs initially did, I would suggest adding a note after the code example explaining that objects created from the `c_char_p` class can only have their value assigned to bytes or integers.
```suggestion
   >>>
   
.. note::
   
   Objects instantiated from :class:`c_char_p` can only have their value set to bytes 
   or integers.
```